addons/l10n_cl/views/report_invoice.xml

```diff
@@ -222,6 +222,14 @@
 
     </template>
 
+    <template id="report_invoice_document_copy" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[contains(@class, 'page')]" position="after">
+            <t t-if="is_copy">
+                <t t-call="l10n_cl.copy_footer"/>
+            </t>
+        </xpath>
+    </template>
```

Comment:
instead of an xpath i think that solution is cleaner 
<img width="527" height="122" alt="image" src="https://github.com/user-attachments/assets/bb0abe54-6bec-4d70-ae18-4c7a354aed74" />

Reply:
(print with cedible is your is_copy)